lib/modules/manager/types.ts

```diff
@@ -112,6 +112,8 @@ export interface Package<T> extends ManagerData<T> {
   currentRawValue?: string;
   major?: { enabled?: boolean };
   prettyDepType?: string;
+
+  prettyVersion?: string;
```

Comment:
```suggestion
  prettyNewVersion?: string;
```

Reply:
looks like we add this on the wrong interface. shouldn't this be on branch update interface? 🤔

Reply:
Yeah, good point. `prettyDepType` can be decided by the manager hence why it's there

Reply:
Moved it to `Upgrade` interface